index.src.html

```diff
@@ -91,11 +92,10 @@
       text: creation URL
     urlPrefix: workers.html
       text: running a worker; url: run a worker
-    urlPrefix: dom.html
-      text: document referrer policy; url: concept-document-referrer-policy; for: Document
   type: interface
     urlPrefix: dom.html
       text: Document
+      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

Comment:
Darn, I must be totally confused. So should this be a `type: dfn`, with the link as `<a lt="document referrer policy">referrer policy</a>` and no `for` attribute?